### ISSUE TITLE
Conflict with IcapBadgeBundle

### ISSUE THREAD

**Issue by Elorfin** [claroline/CoreBundle#1099]
Hi,

There is a conflict between entities when you want to install the BadgeBundle from Icap (which is required by PortfolioBundle).

In the CoreBundle, there is an entity `Claroline\CoreBundle\Entity\Badge\BadgeTranslation` which has the same table name than `Icap\BadgeBundle\Entity\BadgeTranslation`. I think there is the same problem on other Badge entities, but this is the first to occur.

Doctrine becomes weird and doesn't allow any operation to the DB and throws : 

```
[Doctrine\DBAL\Schema\SchemaException]                               
The table with name 'claro.claro_badge_translation' already exists. 
```

Is the namespace `Claroline\CoreBundle\Entity\Badge\` used or can I entirely remove it safely ?

**Comment by maxailloud** [comment]
I don't know how you were able to install portfolio with badge-bundle as a dependency, this update hasn't been merge in master even by mistake as the rest of namespace changes.

Anyway badgeBundle should not be installed for now as the feature is still in the core.

**Comment by Elorfin** [comment]
I have installed a new instance of Claroline using `composer.json.full`.
Then I have had a missing dependency with BadgeBundle.
That's why I've opened the issue iCAPLyon1/PortfolioBundle#42.

I have checked your Portfolio repository and apparently the only branch which contains the dependency to BadgeBundle is `badge-moved`. 
I need to check tomorrow why composer give me this one.

**Comment by maxailloud** [comment]
Yes pretty weird.

No chance I fixed the issue with badge and portfolio few hours before you made it.

**Comment by Elorfin** [comment]
Ok found it.

The dependency has came in with this merge :
https://github.com/iCAPLyon1/PortfolioBundle/commit/99b560a5fa45acc68d45e541c6bb5524c2a3a94f

and has been removed here : 
https://github.com/iCAPLyon1/PortfolioBundle/commit/86ce3672f3838e93e35420ee954557a99cc241ed

So, there is no problem : bug is already fixed ;). I have created my instance before this commit. I will update it.

Thanks for help.
